cns/hnsclient/hnsclient_windows.go

```diff
@@ -67,6 +67,9 @@ const (
 	// aclPriority200 indicates the ACL priority of 200
 	aclPriority200 = 200
 
+	// aclPriority1000 indicates the ACL priority of 1000
+	aclPriority1000 = 1000
```

Comment:
why choose 1000? does this signify higher or lower priority

Reply:
Higher priority for allows, right now we block at 2000 and allow at 200. It allows for the orchestrator to specify a rule that is block (example: 999 block all below) and allow rules to exclude each port/protocol/src/dest address.